nl_server/embeddings.py

```diff
@@ -38,10 +39,6 @@
 # Scores below this are ignored.
 _SV_SCORE_THRESHOLD = 0.5
 
-# If the difference between successive scores exceeds this threshold, then SVs at
-# the lower score and below are ignored.
-_MULTI_SV_SCORE_DIFFERENTIAL = 0.05
-
 _NUM_CANDIDATES_PER_NSPLIT = 3
```

Comment:
nit: given that the other constant was moved to the constants file, why not also this?

Reply:
That is a constants file in shared lib, so only having stuff shared between nl_server and server there.